src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -8,6 +8,7 @@
     <OutputPath>..\..\binaries\</OutputPath>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DisableLockFileFrameworks>true</DisableLockFileFrameworks>
```

Comment:
would it make sense to add a comment with a link to this issue to remember later on why this was added?